canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.scss

```diff
@@ -138,7 +138,10 @@
 			// ReadonlyTable with single row selection is non-interactive. Nothing should happen on hover
 			background-color: $layer-hover-01;
 		}
-
+		&:focus {
```

Comment:
What is this for?  This might need to go under expression builder only and not the default for all tables.